### PR TITLE
KOA-4773 Add compose platform docs

### PR DESCRIPTION
**Pull request by marianeum**
In preparation for releasing compose add support for displaying compose docs. The first compose docs (Text) are available:
screencapture-0-0-0-0-8080-components-text-2021-12-15-10_44_09

The maven central link doesn't work yet, as we haven't published yet, but once we have it'll work.

Remember to include the following changes:
+ [x] [Links platform metadata](https://github.com/Skyscanner/backpack-docs/blob/main/docs/src/layouts/links.js)

### CODE REVIEW

**Review by olliecurtis — CHANGES_REQUESTED** [review]
Code looks good to me! However checking the deployed version the maven and github links give errors when you click on them.

Seems there is no maven build for compose yet? and the github url looks to point to a dead end

Screenshot 2021-12-15 at 12 14 03

**Comment by marianeum** [comment]
Yep as I mentioned above: The maven central link doesn't work yet, as we haven't published yet, but once we have it'll work.

So it points to the right package we'll be publishing on.

**Comment by olliecurtis** [comment]
> Yep as I mentioned above: The maven central link doesn't work yet, as we haven't published yet, but once we have it'll work.
> 
> So it points to the right package we'll be publishing on.

ooop sorry missed that under the image 🤦 sorry about that! Does the same apply for the github link?

**Comment by marianeum** [comment]
Ah sorry I missed this! That is broken indeed, thanks :)

File: docs/src/components/MarkdownPage/AdditionalLinks.js [hunk 1]
```diff
@@ -174,6 +203,20 @@ const AdditionalLinks = (props: Props) => {
           </BpkLink>
         )}
 
+        {/* Android compose GitHub link */}
+        {platform && platform === PLATFORMS.compose && githubPath && (
+          <BpkLink
+            href={`https://github.com/Skyscanner/backpack-android/tree/main/backpack-compose/src/main/java/net/skyscanner/backpack/compose/${githubPath}`}
```
Comment (olliecurtis) [inline docs/src/components/MarkdownPage/AdditionalLinks.js:209]: Looks like the path is `kotlin` and not `java` in the path

```suggestion
            href={`https://github.com/Skyscanner/backpack-android/tree/main/backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/${githubPath}`}
```

**Review by olliecurtis — APPROVED** [review]

**Pull request merged**